core/http/api/v1/oauthserver/routes.go

```diff
@@ -27,22 +27,22 @@ const (
 	AccessTokenPath = "/access_token"
 )
 
-func (api *API) RegisterRoute(engine *gin.Engine) {
+func (a *API) RegisterRoute(engine *gin.Engine) {
 	apiGroup := engine.Group(BasicPath)
 
 	var routes = route.Routes{
 		{
 			Pattern:     AuthorizePath,
 			Method:      http.MethodGet,
-			HandlerFunc: api.HandleAuthorizationGetReq,
+			HandlerFunc: a.HandleAuthorizationGetReq,
```

Comment:
`api` is fine, no need to name it to `a`

Reply:
sync with other packages